ipaserver/plugins/idviews.py

```diff
@@ -1128,6 +1162,15 @@ class idoverrideuser_find(baseidoverride_find):
     msg_summary = ngettext('%(count)d User ID override matched',
                            '%(count)d User ID overrides matched', 0)
 
+    def pre_callback(self, ldap, filter, attrs_list, base_dn, scope, *args,
+                     **options):
+        result = super(idoverrideuser_find, self).pre_callback(
+            ldap, filter, attrs_list, base_dn, scope, *args, **options
+        )
+        filter, base_dn, scope = result
+        filter = self.obj.filter_for_anchor(ldap, filter, options, 'user')
+        return filter, base_dn, scope
```

Comment:
Should we start moving away from using filter as a variable name since it shadows a built-in? This is spread all over and there have been various attempts to work around this (sfilter, search_filter, filterstr, etc).

Reply:
That would be good, however I don't know if any code expects a specific argument name. Is filter always passed by position or sometimes passed by name?

Reply:
I just meant in this case. It isn't a show-stopper, it matches existing behavior.